src/GraphQL/Builders/FieldBuilder.cs

```diff
@@ -132,6 +132,15 @@ protected FieldBuilder(FieldType fieldType)
             return this;
         }
 
+        /// <summary>
+        /// Sets the input resolver for the field.
+        /// </summary>
+        public virtual FieldBuilder<TSourceType, TReturnType> ParseValue(Func<object?, object?> parseValue)
+        {
+            FieldType.ParseValue = parseValue;
+            return this;
+        }
```

Comment:
What should the signature of this method be?  The following might be better, although then is only useful for validation:

`ParseValue(Func<TReturnType?, TReturnType?> parseValue)`

Or perhaps it should be this:

`ParseValue(Func<object?, TReturnType?> parseValue)`